agent/internal/client/base_client.go

```diff
@@ -573,25 +573,6 @@ func (c *baseCommunicator) SendLogMessages(ctx context.Context, taskData TaskDat
 	return nil
 }
 
-// SendTaskResults posts a task's results, used by the attach results operations.
-func (c *baseCommunicator) SendTaskResults(ctx context.Context, taskData TaskData, r *task.LocalTestResults) error {
```

Comment:
Should we also remove `SendTestLog` from the communicators and from the backend?

Reply:
Not yet, I have https://jira.mongodb.org/browse/EVG-19053 for that work.